src/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -185,8 +185,13 @@ propt::resultt satcheck_minisat2_baset<T>::do_prop_solve()
       bool has_false=false;
 
       forall_literals(it, assumptions)
+      {
         if(it->is_false())
-          has_false=true;
+        {
+          has_false = true;
+          break;
+        }
+      }
```

Comment:
May I ask for a change with more churn, likely in a separate commit: the `if(has_false)` below could be removed (and thus one level of indent killed) by just copying its body right here plus
```
status=statust::UNSAT;
return resultt::P_UNSATISFIABLE;
```